mono/mini/Makefile.am.in

```diff
@@ -34,6 +34,52 @@ sgen_libs = \
 	$(monodir)/mono/utils/libmonoutils.la \
 	$(glib_libs)
 
+if ENABLE_LLVM
+# We need to define
+#    LLVM_CFLAGS
+#    LLVM_CXXFLAGS
+#    LLVM_LIBS
+#    LLVM_LDFLAGS
+
+if INTERNAL_LLVM 
```

Comment:
Fix indentation.

Reply:
The rest of the Makefile.am files do not indent the content of the blocks, so remove the `  ` preceding each line.

Reply:
Fixed.